homeassistant/components/camera/netatmo.py

```diff
@@ -76,7 +76,7 @@ def camera_image(self):
                     self._localurl), timeout=10)
             else:
                 response = requests.get('{0}/live/snapshot_720.jpg'.format(
-                    self._vpnurl), timeout=10)
+                    self._vpnurl), timeout=10, verify=False)
```

Comment:
Agree but in this case we are modifying the configuration of a component with a minor update, I think this is something we try to avoid.